libcalico-go/lib/backend/model/keys.go

```diff
@@ -354,6 +350,16 @@ func keyFromDefaultPathInner(path string, parts []string) Key {
 				case "labels":
 					return ProfileLabelsKey{ProfileKey: ProfileKey{pk}}
 				}
+			default:
+				// Policy key of form /calico/v1/policy/<Kind>/<Namespace>/<Name>
```

Comment:
This is a breaking change on the typha/felix API, during upgrade backlevel felix talking to uplevel typha will see no policies and vice versa.